packages/service-library/src/servicelib/db_async_engine.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from fastapi import FastAPI
+from settings_library.postgres import PostgresSettings
+from simcore_postgres_database.utils_aiosqlalchemy import (
+    get_pg_engine_stateinfo,
+    raise_if_migration_not_ready,
+)
+from sqlalchemy.ext.asyncio import AsyncEngine, create_async_engine
+from tenacity import retry
+
+from .retry_policies import PostgresRetryPolicyUponInitialization
+
+logger = logging.getLogger(__name__)
+
+
+@retry(**PostgresRetryPolicyUponInitialization(logger).kwargs)
+async def connect_to_db(app: FastAPI) -> None:
+    logger.debug("Connecting db ...")
```

Comment:
:+1: very nice.

minor: But I do not see any test at the service-library level. that could be nice provided it's quite a central piece now.Maybe add an issue.

Reply:
yes you are right! i will create an issue for that and assign myself